app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
```

Comment:
@pablocubico One way is to set `font-size: 62.5%` on <html> - this makes any font below to act somewhat more familiar: 1.2rem == 12px, 1.4rem == 14px, and so on. Anyway, I think after moving to BS3 we can change them, it's not a big problem.